### PR TITLE
Lots of refactoring, restructuring, etc. (housekeeping)

### PR DESCRIPTION
**Pull request by meredith-myers**
@rogen-code @cinziaborello @JacksonTodd93 
I moved some files around and refactored a couple things on both the front-end and back-end. Basically just some housekeeping from the past few days to make sure everything's good to go.

### CODE REVIEW

File: client/src/Gallery.jsx [hunk 1]
```diff
@@ -158,7 +158,8 @@ class Gallery extends React.Component {
                   <GalleryImage
                     image={image}
                     length={batch.length}
-                    key={image.id}
+                    batch={batch}
+                    /*key={image.id}*/
```
Comment (cinzia-sss) [inline client/src/Gallery.jsx:162]: why did you remove the key? doesn't React give you a warning in console now?

File: database/proxyStressTesting.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  phases:
+    - duration: 60
+      arrivalRate: 5
+      name: Warm up
+    - duration: 120
+      arrivalRate: 5
+      rampTo: 50
+      name: Ramp up load
+    - duration: 600
+      arrivalRate: 50
```
Comment (cinzia-sss) [inline database/proxyStressTesting.yml:12]: why is your proxy test in your service repo? shouldn't it be in the proxy repo? 
also, doesn't this mean that it creates only 50 RPS for six minutes? is this your final test?

File: package.json [hunk 3]
```diff
@@ -10,11 +10,12 @@
...
     "start": "node server/index.js",
-    "seed": "node server/seed.js --entries 10000000 --output server/listingImageData.csv",
-    "seed:test": "node server/seed.js --entries 10 --output server/testListingImageData.csv",
+    "seed": "node database/seed.js --entries 10000000 --output server/listingImageData.csv",
+    "seed:test": "node database/seed.js --entries 10 --output server/testListingImageData.csv",
     "test": "jest",
     "build:dev": "webpack -d --watch",
-    "build": "webpack -d"
+    "build": "webpack -d",
+    "stress": "artillery run database/serviceStressTesting.yml"
```
Comment (cinzia-sss) [inline package.json:18]: this is a good idea, didn't think about it 🙌

File: package.json [hunk 4]
```diff
@@ -36,7 +37,9 @@
     "identity-obj-proxy": "^3.0.0",
     "jest-sinon": "^1.0.4",
     "jquery": "^3.5.1",
-    "nodemon": "^2.0.4",
+    "newrelic": "^6.13.0",
+    "pg": "^8.3.3",
+    "pg-promise": "^10.6.1",
```
Comment (cinzia-sss) [inline package.json:42]: are you testing both drivers? does one of them perform better?

File: database/serviceStressTesting.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  phases:
+    - duration: 60
+      arrivalRate: 5
+      name: Warm up
+    - duration: 120
+      arrivalRate: 5
+      rampTo: 50
+      name: Ramp up load
+    - duration: 600
+      arrivalRate: 50
```
Comment (cinzia-sss) [inline database/serviceStressTesting.yml:12]: same comment: doesn't this mean that it creates only 50 RPS for six minutes? is this your final test?

**Review by cinziaborello — COMMENTED** [review]
Hi @meredith-myers, don't prioritize this over deployment, but there are a couple of things here that I'm not sure why you made a specific choice about it and would love to get more context 🙂
